src/molecules/tables/tables.stories.tsx

```diff
@@ -379,3 +379,51 @@ export const DetailedFilter: StoryObj<typeof Table> = {
     layout: 'centered',
   },
 };
+
+const subContentDialogAction: TableAction = {
+  cta: 'New Dialog',
+  type: 'SubContentDialog',
+  description: 'Add New Record',
+  content: <div>İçerik</div>,
```

Comment:
If we going to open this up as open source library, how do you feel about using English as example language?